package.json

```diff
@@ -1,26 +1,27 @@
-{
-  "name": "metaversefile",
-  "version": "0.0.32",
-  "description": "",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/webaverse/metaversefile.git"
-  },
-  "author": "Avaer Kazmer <a@modules.io>",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/webaverse/metaversefile/issues"
-  },
-  "homepage": "https://github.com/webaverse/metaversefile#readme",
-  "dependencies": {
-    "@babel/core": "^7.15.0",
-    "@babel/preset-react": "^7.14.5",
-    "mime-types": "^2.1.32",
-    "node-fetch": "^2.6.1",
-    "pako": "^2.0.4"
-  }
-}
+{

```

Comment:
Why the change to the line endings? Was this necessary to get windows to work? It isn't explained in the PR.